apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -14,49 +14,87 @@ import TooltipOverlay, {coordinatesProvider} from '../templates/TooltipOverlay';
 var GAME_WIDTH = gameLabConstants.GAME_WIDTH;
 var GAME_HEIGHT = gameLabConstants.GAME_HEIGHT;
 
-var GameLabVisualizationColumn = function (props) {
-  var divGameLabStyle = {
-    width: GAME_WIDTH,
-    height: GAME_HEIGHT
-  };
-  return (
-    <span>
-      <ProtectedVisualizationDiv>
-        <div id="divGameLab" style={divGameLabStyle} tabIndex="1">
-        </div>
```

Comment:
Initially read this as storing (data-wise) the coordinates in app-space, maybe `Cache app-space mouse coordinates`